packages/graph-engine/src/nodes/color/distance.ts

```diff
@@ -1,49 +1,68 @@
-import { Color } from "../../types.js";
-import { ColorSchema, NumberSchema } from "../../schemas/index.js";
-import { INodeDefinition, ToInput, ToOutput } from "../../index.js";
+
+import { ColorSchema, NumberSchema, StringSchema } from "../../schemas/index.js";
+import { INodeDefinition} from "../../index.js";
 import { Node } from "../../programmatic/node.js";
-import chroma from "chroma-js";
+import Color from "colorjs.io";
+
+export const colorSpaces = [
+  "Lab",
+  "ICtCp",
+  "Jzazbz",
+] as const;
+
+export type ColorSpace = typeof colorSpaces[number];
+
 export default class NodeDefinition extends Node {
-  static title = "Color Distance";
+  static title = "Distance";
   static type = "studio.tokens.color.distance";
   static description =
     "Distance node allows you to calculate the distance between two colors.";
 
 
-  declare inputs: ToInput<{
```

Comment:
Why did you remove this ?

Reply:
isn't this overhead with the new version?

Reply:
This declare is still missing